### PR TITLE
CF: Update FTC settlement caution flag plain language mappings #10474

### PR DESCRIPTION
**Pull request by zurbergram**
## Description

https://github.com/department-of-veterans-affairs/va.gov-team/issues/10474

## Acceptance criteria
- [ ]

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Review by shawkey-daniel — APPROVED** [review]
LGTM

File: db/seeds/03_rules.rb [hunk 1]
```diff
@@ -131,6 +131,18 @@ def rule_id(rule_results, object)
                predicate: 'reason',
                object: 'States Attorney General Filed Suit for Deceptive Action',
                priority: 2),
+      Rule.new(rule_name: CautionFlag.name,
+               matcher: Rule::MATCHERS[:has],
+               subject: nil,
+               predicate: 'reason',
+               object: 'The Federal Trade Commission (FTC) has reached a settlement with this school',
+               priority: 2),
+      Rule.new(rule_name: CautionFlag.name,
```
Comment (erluti) [inline db/seeds/03_rules.rb:140]: These two rules look identical to me.  Is this intentional?

Comment (zurbergram) [reply db/seeds/03_rules.rb:140]: one object field has a period on the end the other doesn't

Comment (erluti) [reply db/seeds/03_rules.rb:140]: I see that now.  I'm curious what the purpose of that is?

**Review by philipwritescode — APPROVED** [review]

**Review by delli-gatti-michael — APPROVED** [review]

**Comment by zurbergram** [comment]
@erluti The mapping done in https://github.com/department-of-veterans-affairs/gibct-data-service/blob/master/app/models/caution_flag.rb#L9 does exact matches using https://github.com/ulfurinn/wongi-engine

**Pull request merged**